### ISSUE TITLE
Give more users the option to kick logged in players

### ISSUE THREAD

**Issue by mcepok** [teheru/nczone#8]
* We already have a kick function at controller/api.php -> player_logout()
* Currently there is a single mod permission acl::m_zone_login_players in order to do this
* We need another user permission acl::u_zone_login_players (or something) for normal users in order to be able to kick other users under certain conditions
* For now, this condition this condition should be a certain idle time of the user to be kicked
* The actual amount of time should be configurable in the ACP

**Comment by Des-** [comment]
Are you sure that we want to implement this without checking for nC-Zone reminder activities?

**Comment by mcepok** [comment]
Yes, as you're basically the only one using it. :stuck_out_tongue: 
We need some possibility to connect an external tool with the zone like an API key, so the reminder can tell the zone someone is active. That would be a bigger project though.
